system/Honeypot/Exceptions/HoneypotException.php

```diff
@@ -16,22 +16,22 @@
 
 class HoneypotException extends ConfigException implements ExceptionInterface
 {
-    public static function forNoTemplate()
+    public static function forNoTemplate(): self
```

Comment:
This is a breaking change.
https://3v4l.org/GGS0f
Please use doc comment.

Reply:
Done.